files/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>WoMakers Code</title>
+</head>
+<body>
+	<nav class="wm-nav">
+		<h1 class="wm-nav__logo"></h1>
+		<ul class="wm-nav__menu">
+			<li class="wm-nav__item"><a href="#">Home</a></li>
+			<li class="wm-nav__item"><a href="#">Serviços</a></li>
+			<li class="wm-nav__item"><a href="#"></a>Portfolio</a></li>
+			<li class="wm-nav__item"><a href="#">Sobre Mim</a></li>
+			<li class="wm-nav__item"><a href="#">Contato</a></li>
+		</ul>
+	</nav>
+	<header class="wm-header">
+		<h2 class="wm-header__title">Bem Vind@</h2>
+		<h3 class="wm-header__subtitle">Carabitur iaculis</h3>
+		<p class="wm-header__text">texto texto texto</p>
+	</header>
+	<div class="wm-servicos" id="servicos">
+		<h1 class="wm-servicos__title">Serviços</h1>
+		<h3 class="wm-servicos__subtitle">texto texto texto</h3>
+		<div class="wm-sevicos__group">
+			<div class="wm-servico">
+				<img src="#" alt="websites">
+				<h4 class="wm-servico__title">Websites</h4>
+				<p class="wm-servico__description">texto texto texto</p>
+			</div>
+			<div class="wm-servico">
+				<img src="#" alt="templates">
+				<h4 class="wm-servico__title">Templates</h4>
+				<p class="wm-servico__description">texto texto texto</p>
+			</div>
+			<div class="wm-servico">
+				<img src="#" alt="Temas WordPress">
+				<h4 class="wm-servico__title">Temas WordPress</h4>
+				<p class="wm-servico__description">texto texto texto</p>
+			</div>
+		</div>
+	</div>
+	<div class="wm-jobs">
+		<h1 class="wm-jobs__title">Meus Trabalhos</h1>
+		<h3 class="wm-jobs__subtitle">texto texto texto</h3>
+		<div class="wm-jobs__group">
+			<ul class="wm-jobs__filters">
+				<li class="wm-jobs__filter"><button>Tudo</button></li>
+				<li class="wm-jobs__filter"><button>Website</button></li>
+				<li class="wm-jobs__filter"><button>Template</button></li>
+				<li class="wm-jobs__filter"><button>Tema</button></li>
+			</ul>
+			<div class="wm-portfolio">
+				<img class="wm-portfolio__img" src="#" alt="job">
+				<img class="wm-portfolio__img" src="#" alt="job">
+				<img class="wm-portfolio__img" src="#" alt="job">
+			</div>
+		</div>
+	</div>
+	<div class="wm-about">
+		<h1 class="wm-about__title">Quem sou eu</h1>
+		<h3 class="wm-about__subtitle"></h3>
```

Comment:
Mesma sugestão acima:

```suggestion
<h2 class="wm-about__subtitle"></h2>
```